packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -408,6 +408,65 @@ const defaultConfigs = {
       }
     ]
   },
+  "USD/bBadger": {
+    type: "expression",
+    // Note: lower-case variables are intermediate, upper-case are configured feeds.
+    expression: `
+      wbtc_usd = mean(WBTC_ETH_SUSHI, WBTC_ETH_UNI) / USDETH;
```

Comment:
Should we define this as a separate config since it is used multiple times?

Reply:
I don't think that's super easy to do right now and would require a bit of a refactor. I'd say leave it like this is fine for now until we have a more generalised way to pull out common strings/vars that can be re-used within feeds.